ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -176,12 +176,14 @@ class KeyValueCommitting private[daml] (
         )
 
       case DamlSubmission.PayloadCase.TRANSACTION_ENTRY =>
-        new ProcessTransactionSubmission(defaultConfig, engine, metrics, inStaticTimeMode)
+        new TransactionCommitter(defaultConfig, engine, metrics, inStaticTimeMode)
           .run(
             entryId,
+            //TODO replace this call with an explicit maxRecordTime from the request once available
```

Comment:
Can you not add this TODO here, please? We already have it in many places and this should be rather captured in our backlog.

Reply:
Done